build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -123,7 +123,7 @@ steps:
       set -e
       APP_ROOT=$(agent.builddirectory)/azuredatastudio-darwin
       APP_NAME="`ls $APP_ROOT | head -n 1`"
-      yarn smoketest --build "$APP_ROOT/$APP_NAME" --screenshots "$(build.artifactstagingdirectory)/smokeshots"
+      yarn smoketest --build "$APP_ROOT/$APP_NAME" --screenshots "$(build.artifactstagingdirectory)/smokeshots" --log "$(build.artifactstagingdirectory)/smokelogs/smoke.log"
```

Comment:
Let's just put this under `$(build.artifactstagingdirectory)/logs/linux-x64` so it's combined with the rest of the logs from test runs and is separated out by platform in case we end up running smoke tests on other platforms in the future.